lib/timeline/timeline.dart

```diff
@@ -46,9 +46,29 @@ class TimelineScreen extends Screen {
   PButton resumeButton;
 
   @override
-  void createContent(Framework framework, CoreElement mainDiv) {
+  void createContent(Framework framework, CoreElement mainDiv) async {
```

Comment:
I would be a little concerned about making this method Async. I would expect a reasonable contract is that this method is not async.